frontend/src/pages/Konva.js

```diff
@@ -0,0 +1,158 @@
+import React, {useState} from 'react';
+import {Stage, Layer, Rect, Circle} from 'react-konva';
+
+export default function Konva() {
+    const [rects, setRects] = useState([]);
+    const [circles, setCircles] = useState([])
+    const [rectId, setRectId] = useState(1)
+    const [circleId, setCircleId] = useState(1)
+
+    const addRect = () => {
+        setRects([...rects, {
+            id: rectId.toString(),
+            x: 60,
+            y: 60,
+            width: 30,
+            height: 30,
+            fill: '#C9E1F2',
+            stroke: '#000',
+            strokeWidth: 1,
+            shadowBlur: 10,
+        }
+        ])
+        setRectId(rectId + 1);
+    }
+
+    const addCircle = () => {
+        setCircles([...circles, {
+            id: circleId.toString(),
+            x: 60,
+            y: 60,
+            radius: 15,
+            fill: '#C9E1F2',
+            stroke: '#000',
+            strokeWidth: 1,
+            shadowBlur: 10
+        }
+        ])
+        setCircleId(circleId + 1);
+    }
+
+
+    const handleDragStart = (e) => {
+        const id = e.target.id();
+        const x = e.target.x();
+        const y = e.target.y();
+
+        console.log(
+            'ID: ' + id + '  ' +
+            'X: ' + x + '  ' +
+            'Y: ' + y)
+    }
+
+    const setRectCoordinatesOnDragEnd = (e) => {
+        const id = e.target.id();
+        const x = e.target.x();
+        const y = e.target.y();
+
+        setRects(
+            rects.map((rect) => {
+                if (rect.id === id) {
+                    return {
+                        ...rect,
+                        x: x,
+                        y: y,
+                        isDragging: false,
+                        width: 30,
+                        height: 30,
+                        shadowBlur: 10,
+                        fill: '#C9E1F2', shadowOffsetX: 5, shadowOffsetY: 5
+                    }
+                } else {
+                    return {...rect}
+                }
+            }))
+
+        console.log(
```

Comment:
console.log is super helpful during debugging, but I prefer to not leave it in the code, it has a tendency to litter up the code base and the console.